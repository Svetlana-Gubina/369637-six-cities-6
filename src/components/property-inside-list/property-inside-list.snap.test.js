import React from 'react';
import {render} from '@testing-library/react';
import PropertyInside from './property-inside';

test(`Should PropertyInside render correctly`, () => {
  const goodsList = [`Heating`, `Kitchen`, `Cable TV`];

  const {container} = render(<PropertyInside goods={goodsList} />);
  expect(container).toMatchSnapshot();
});
