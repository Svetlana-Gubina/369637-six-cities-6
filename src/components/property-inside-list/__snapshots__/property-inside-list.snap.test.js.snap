// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PropertyInside render correctly 1`] = `
<div>
  <ul
    class="property__inside-list"
  >
    <li
      class="property__inside-item"
    >
      Heating
    </li>
    <li
      class="property__inside-item"
    >
      Kitchen
    </li>
    <li
      class="property__inside-item"
    >
      Cable TV
    </li>
  </ul>
</div>
`;
